src/services/helpers.ts

```diff
@@ -56,3 +58,36 @@ export const shuffleArray = <T>(array: T[]): T[] => {
   }
   return shuffled
 }
+
+export const getNextExercise = async (
```

Comment:
i think due to the complexity of this fkt, it would be nice to get a short comment